app/loader.cpp

```diff
@@ -112,6 +112,10 @@ void Loader::zoomToProject( QgsQuickMapSettings *mapSettings )
     QgsRectangle layerExtent = mapSettings->mapSettings().layerExtentToOutputExtent( layer, layer->extent() );
     extent.combineExtentWith( layerExtent );
   }
+  if ( extent.isEmpty() )
```

Comment:
Doesn't seem to be related to overflow issue with ActiveLayerPanel to me. If it is not, please, remove it from PR.

Reply:
Yes it isn't related. Sorry I don't know GitHub well :) I already did pull request for this cpp file before. I didn't know GitHub would consider it as a change although I changed QML file only.

Reply:
yes, no worries, seems just like a leftover. However could be nice to unstage/reset the files/changes not related to PR.